cea/optimization/preprocessing/preprocessing_main.py

```diff
@@ -1,6 +1,6 @@
 """
 ============================
-pre-processing algorithm
+Pre-processing algorithm
 ============================
```

Comment:
@Jack-Hawthorne it is not clear why some of the lines  "===============" are erased and some others are not across different scripts. I think they should be erased from everywhere except where they are needed to be read by sphynxk. Would you mind to check for consistency?